workspaces/ui/src/utilities/EndpointUtilities.js

```diff
@@ -0,0 +1,92 @@
+"use strict";
```

Comment:
didn't mean to commit this

Reply:
Was gonna say, we've got this as TS, right? 😅